dockerfile/builder/Dockerfile

```diff
@@ -4,7 +4,7 @@
 FROM ros:dashing
 RUN mkdir -p uros_ws
 WORKDIR uros_ws
-RUN git clone --depth 1 -b feature/mempools_refactor https://github.com/micro-ROS/micro-ros-build.git src/micro-ros-build \
+RUN git clone --depth 1 -b https://github.com/micro-ROS/micro-ros-build.git src/micro-ros-build \
```

Comment:
I think we could use the `microros/base:dashing` once the aforementioned PRs are merged.

Reply:
Done